script/1_0_2current_en_script/Arcueid/Day 2/02_00_ARC02_4B.txt

```diff
@@ -43,8 +43,8 @@ It was a major victory in terms of a brother's dignity.
 I'm on my way back to my room when Kohaku-san comes to talk to me.
 Just like this morning, she hides herself in a corner of the corridor and talks in private.
 <Page13>
-"Come on. That's no good, you should have come home on time today at least.
-No matter how you look at it, this is all because you took so long to come home after saying that you would keep your promise for a day at least."
+"Come on. That's no good, you should have come home on time today, at least.
+No matter how you look at it, this is all because you took so long to come home after saying that you would keep your promise for a day, at least."
```

Comment:
Ok yeah this line is translated wrong, pretty sure Gekimon fixed this, you should merge master into this PR to clear up this and I think one other case where you're tweaking a dead line